evap/evaluation/models.py

```diff
@@ -421,8 +421,8 @@ def publish(self):
 
     @transition(field=state, source='published', target='reviewed')
     def unpublish(self):
-        from evap.results.tools import get_results_cache_key
-        caches['results'].delete(get_results_cache_key(self))
+        from evap.results.tools import get_collect_results_cache_key
```

Comment:
maybe `collected_results`?

Reply:
i was trying to say "this is the cache key of the method `collect_results`". not sure whether that makes sense, what do you think?

Reply:
kind of :)
i think naming is weird both ways so don't bother.